media/js/firefox/whatsnew/whatsnew-id.js

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ (function() {
+     'use strict';
+
+     var client = Mozilla.Client;
+     var form = new Mozilla.SendYourself('download-firefox-rocket');
+     form.init();
+
+     function checkUpToDate() {
+         // bug 1419573 - only show "Your Firefox is up to date" if it's the latest version.
+         if (client.isFirefoxDesktop) {
+             client.getFirefoxDetails(function(data) {
+                 if (data.isUpToDate) {
+                     document.querySelector('.c-page-header').classList.add('is-up-to-date');
+                 }
+             });
+         }
+     }
+
+     Mozilla.UITour.ping(function() {
```

Comment:
This is throwing an error `TypeError: Mozilla.UITour is undefined`. Looks like you might need to include "js/base/uitour-lib.js"` in the bundle.